docs/gathered-data.md

```diff
@@ -13,1040 +13,1878 @@ archives are packaged as `.tar.gz` files in `/var/lib/insights-operator`.
+
+### Released version
+- 4.10.0
+
+### Backported versions
+None
+
+### Changes
+None
 
 
 ## HostSubnet
 
-collects HostSubnet information
+Collects HostSubnet information.
+
+### API Reference
+- https://github.com/openshift/client-go/blob/master/network/clientset/versioned/typed/network/v1/hostsubnet.go
+- https://docs.openshift.com/container-platform/4.3/rest_api/index.html#hostsubnet-v1-network-openshift-io
 
-The Kubernetes api https://github.com/openshift/client-go/blob/master/network/clientset/versioned/typed/network/v1/hostsubnet.go
-Response see https://docs.openshift.com/container-platform/4.3/rest_api/index.html#hostsubnet-v1-network-openshift-io
+### Sample data
+- [docs/insights-archive-sample/config/hostsubnet](./insights-archive-sample/config/hostsubnet)
+
+### Location in archive
+- `config/hostsubnet/{name}.json`
+
+### Config ID
+`clusterconfig/host_subnets`
 
-* Location in archive: config/hostsubnet/
-* Id in config: clusterconfig/host_subnets
-* Since versions:
-  - 4.4.29+
-  - 4.5.15+
-  - 4.6+
+### Released version
+- 4.6.0
+
+### Backported versions
+- 4.4.29+
+- 4.5.15+
+
+### Changes
+None
 
 
 ## ImageStreamsOfNamespace
 
-creates a gathering closure which collects image streams from the provided namespace
+Creates a gathering closure which collects image streams from the provided namespace
+
 Params is of type GatherImageStreamsOfNamespaceParams:
-  - namespace string - namespace from which to collect image streams
+- namespace string - namespace from which to collect image streams
+
+### API Reference
+- https://docs.openshift.com/container-platform/4.7/rest_api/image_apis/imagestream-image-openshift-io-v1.html#apisimage-openshift-iov1namespacesnamespaceimagestreams
+
+### Sample data
+- [docs/insights-archive-sample/conditional/namespaces/openshift-cluster-samples-operator/imagestreams/example.json](./insights-archive-sample/conditional/namespaces/openshift-cluster-samples-operator/imagestreams/example.json)
+
+### Location in archive
+| Version   | Path														|
+| --------- | --------------------------------------------------------	|
+| >= 4.9.0  | conditional/namespaces/{namespace}/imagestreams/{name}.json |
+
+### Config ID
+`conditional/image_streams_of_namespace`
 
-API reference:
+### Released version
+- 4.9.0
 
-	https://docs.openshift.com/container-platform/4.7/rest_api/image_apis/imagestream-image-openshift-io-v1.html#apisimage-openshift-iov1namespacesnamespaceimagestreams
+### Backported versions
+None
 
-* Location in archive: conditional/namespaces/{namespace}/imagestreams/{name}
-* Id in config: conditional/image_streams_of_namespace
-* Since versions:
-  - 4.9+
+### Changes
+None
 
 
 ## InstallPlans
 
-collects Top x InstallPlans from all openshift namespaces.
-Because InstallPlans have unique generated names, it groups them by namespace and the "template"
-for name generation from field generateName.
+Collects top X InstallPlans from all openshift namespaces. Because InstallPlans have
```

Comment:
Begs the question what is "X"...

Reply:
And it should be `OpenShift` when I am seeing it....

Reply:
I guess that the original idea for the person that wrote it was to use `X`, since the value of the limit of items could change in code, but... it isn't helpful when we are reading.

For the `Openshift` I guess that is better if we use `openshift-* namespace`, like:

https://github.com/openshift/insights-operator/blob/1f10ef9673a311c232d51aa68dbac31cfb56e65b/pkg/gatherers/clusterconfig/install_plans.go#L81